docs/ntppool/en/use.html

```diff
@@ -73,23 +99,23 @@ <h3 id="use">How do I use pool.ntp.org?</h3>
 <div class="block">
 	<h3 id="notes">Additional Notes</h3>
 
-        <p><span class="hook">Consider if the NTP Pool is appropriate
-        for your use</span>. If business, organization or human life
-        depends on having correct time or can be harmed by it being
-        wrong, you shouldn't "just get it off the internet". The NTP
-        Pool is generally very high quality, but it is a service run
-        by volunteers in their spare time. Please talk to your
-        equipment and service vendors about getting local and reliable
-        service setup for you. See also our <a href="/tos.html">terms
-        of service</a>.
-
-        We recommend time servers from
-        <a href="http://www.meinbergglobal.com/english/products/ntp-time-server.htm">Meinberg</a>,
-        but you can also find time servers from
-        <a href="http://www.endruntechnologies.com/NTP-Servers/gps-cdma-ntp.htm">End Run</a>,
-        <a href="http://spectracom.com/products-services/precision-timing#anchor-2172">Spectracom</a>
-        and many others.
-        </p>
+		<p><span class="hook">Consider if the NTP Pool is appropriate
+		for your use</span>. If business, organization or human life
+		depends on having correct time or can be harmed by it being
+		wrong, you shouldn't "just get it off the internet". The NTP
+		Pool is generally very high quality, but it is a service run
+		by volunteers in their spare time. Please talk to your
+		equipment and service vendors about getting local and reliable
+		service setup for you. See also our <a href="/tos.html">terms
+		of service</a>.
+
+		We recommend time servers from
+		<a href="http://www.meinbergglobal.com/english/products/ntp-time-server.htm">Meinberg</a>,
+		but you can also find time servers from
+		<a href="https://endruntechnologies.com/products/ntp-time-servers">End Run</a>,
+		<a href="https://www.orolia.com/solution/timing-and-synchronization/">Orolia</a>
+		and many others.
+		</p>
```

Comment:
I still think it would be better if this were restricted to just substantive changes and the whitespace left for another PR.

Reply:
@paulgear, in general I agree with you and I'm aware of potential merge conflicts due to whitespace changes. However, I'm a big supporter of strict whitespace standards in documents, be it code or documentation.

Above section was introduced in 8c5e7b2b by @abh using space indentation, while the rest of the document used tab indentation. Probably an oversight and not worth fixing after the commit is already made. But since I'm editing this document, I straightened out that formatting inconsistency.

I also have a hard time telling my editor to use space and tab indentation together in one document. It's one or the other.

But, since I'm a first time contributor here, I'm open to discussion.